bin/test/main.ml

```diff
@@ -24,6 +24,8 @@ module Log = (val Logs.src_log src : Logs.LOG)
 
 let (/) = Filename.concat
 
+let (>>-) x f = (|>) x (List.iter f)
```

Comment:
Why not defining this like this `List.iter f x` ?
Also, I'm not fan of defining this as an operator as you had to define it twice in 2 modules.
I agree having the function as the last argument is slightly better, `abc >>- (fun x ->` could be written `abc |> List.iter (fun x ->`. What do you think?

Reply:
The definition I've given reads in the way I think about it, so I think it's easier :slightly_smiling_face:

I think in files where there are lots of occurrences of this pattern it makes sense to define an infix operator for it, but I don't particularly mind `|> List.iter` as an alternative.  I also think it's fairly natural to need to redefine this sort of helper in multiple files, but again I don't particularly mind if you think otherwise.

Reply:
https://github.com/realworldocaml/mdx/pull/173/commits/865fe919e93547601fe88adc01ee546aa056d538 rewrites the `>>-` definitions as `List.iter f x` for simplicity :slightly_smiling_face: